Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -3516,7 +3474,2305 @@
+                  "type": 3,
+                  "content": {
+                    "version": "KqlItem/1.0",
+                    "query": "let appServerList = dynamic([{CpuMemApplicationServer}]);\r\nlet workProcessType = '{CPUWorkProcessType}';\r\nlet maxTimestamp = SapNetweaver_ABAPGetWPTable_CL\r\n| where SID_s == '{CpuMemSID}'\r\n| extend instanceNr_s = tostring(toint(instanceNr_d))\r\n| extend instanceNr_s = iff(strlen(instanceNr_s) == 2, instanceNr_s, strcat( \"0\", instanceNr_s))\r\n| extend appServer = strcat(hostname_s, \"_\", instanceNr_s)\r\n| where array_length(appServerList) == 0 or appServer in (appServerList)\r\n| where Typ_s == workProcessType\r\n| where Status_s == 'Run' or Status_s == 'On Hold'\r\n| summarize max(serverTimestamp_t);\r\n\r\nSapNetweaver_ABAPGetWPTable_CL\r\n| where serverTimestamp_t == toscalar(maxTimestamp)\r\n| where SID_s == '{CpuMemSID}'\r\n| extend instanceNr_s = tostring(toint(instanceNr_d))\r\n| extend instanceNr_s = iff(strlen(instanceNr_s) == 2, instanceNr_s, strcat( \"0\", instanceNr_s))\r\n| extend appServer = strcat(hostname_s, \"_\", instanceNr_s)\r\n| where array_length(appServerList) == 0 or appServer in (appServerList)\r\n| where Typ_s == workProcessType\r\n| where Status_s == 'Run' or Status_s == 'On-Hold'\r\n| extend ApplicationServerInstance = strcat(hostname_s, \"_\", SID_s,\"_\", instanceNr_s), ProcessNumber = No_d, ProcessType = Typ_s, ProcessId = Pid_d, Status = Status_s, Reason = Reason_s, Error = Err_s, CPU = Cpu_s, Time = Time_s, Program = Program_s, User = User_s, Client = Client_s\r\n| project ApplicationServerInstance, ProcessNumber, ProcessType, ProcessId, Status, Reason, Error , CPU,Time, Program, User, Client\r\n",
+                    "size": 3,
+                    "title": "Active Work Process (SM66)",
+                    "queryType": 0,
+                    "resourceType": "microsoft.operationalinsights/workspaces",
+                    "visualization": "table",
+                    "chartSettings": {
+                      "xAxis": "timestamp_t",
+                      "yAxis": [
+                        "FREE_MEM_MB_d",
+                        "ACT_WPS_d"
+                      ],
+                      "seriesLabelSettings": [
+                        {
+                          "seriesName": "ACT_DIA_d",
+                          "label": "Active Dialog  WPs"
+                        },
+                        {
+                          "seriesName": "ACT_WPS_d",
+                          "label": "Active WPs​"
+                        },
+                        {
+                          "seriesName": "TRFC_FREE_d",
+                          "label": "Available RFC WPs"
+                        },
+                        {
+                          "seriesName": "FREE_MEM_MB_d",
+                          "label": "Free WPs"
+                        },
+                        {
+                          "seriesName": "FREE_MEM_MB_INC_FS_d",
```

Comment:
copy and paste artifact?  this seems like memory series that does not belong in this chart